Utils/CommonDriver.cs

```diff
@@ -0,0 +1,14 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MarsSpecFlow_HannahN.Utils
+{
+    internal class CommonDriver
+    {
+        public IWebDriver driver;
```

Comment:
you have not finished implementing this

Reply:
please make use of Hooks , you might need to create the Common driver object there!
Please see
https://codebun.com/how-to-implement-hooks-in-specflow-c/